shared/chat/conversation/input-area/normal/platform-input.native.tsx

```diff
@@ -171,32 +174,38 @@ class _PlatformInput extends PureComponent<PlatformInputPropsInternal, State> {
               openExplodingPicker={() => this._toggleShowingMenu('exploding')}
             />
           )}
-          <Kb.PlainInput
-            autoCorrect={true}
-            autoCapitalize="sentences"
-            disabled={
-              // Auto generated from flowToTs. Please clean me!
-              this.props.cannotWrite !== null && this.props.cannotWrite !== undefined
-                ? this.props.cannotWrite
-                : false
-            }
-            placeholder={hintText}
-            multiline={true}
-            onBlur={this.props.onBlur}
-            onFocus={this.props.onFocus}
-            // TODO: Call onCancelQuoting on text change or selection
-            // change to match desktop.
-            onChangeText={this._onChangeText}
-            onSelectionChange={this.props.onSelectionChange}
-            ref={this._inputSetRef}
-            style={styles.input}
-            textType="Body"
-            rowsMax={Styles.dimensionHeight < 600 ? 5 : 9}
-            rowsMin={1}
-          />
+          {
```

Comment:
???

Reply:
Sorry I was hiding this whole thing at one point and just left that in there.